crates/types/src/config/bifrost.rs

```diff
@@ -36,7 +36,7 @@ use super::{
 pub struct BifrostOptions {
     /// # The default kind of loglet to be used
     ///
-    /// Default: Local
+    /// Default: Replicated
     pub default_provider: ProviderKind,
```

Comment:
bonus points if we can follow up with a PR that skips serialization if it's the default value already. Same for metadata.